README.md

```diff
@@ -21,5 +21,13 @@ Check full features and entities involved on the [wiki home page](https://github
 8. Create database: `rake db:create`
 9. Run migrations: `rake db:migrate`
 10. Seed database with initial data: `rake db:seed`
+This creaete five users with 
```

Comment:
We are adding a lot of data with `rake db:seed` which includes the data of 3 Locations `{Location 1, Location 2, Location 3}` and also 14 Nooks `{Nice Office 1..Nook 14}`